packages/devtools/package.json

```diff
@@ -0,0 +1,23 @@
+{
+	"name": "@wordpress/devtools",
```

Comment:
I'm really bad at finding names so I'll leave this to others. 

Thinking that people could confuse `devtools` with actual dev tools not used in the code. Do you think we should try to find a name that indicates that it's a runtime dependency? I'm not certain there's a better name, just asking.

Reply:
Me as well, I asked for help: https://wordpress.slack.com/archives/C5UNMSU4R/p1527080592000279.